### DISCUSSION TITLE
No recipes in the crafting tab to start with

### DISCUSSION THREAD

**Discussion by R0gu3P403n1x** [AngryBeaver/beavers-crafting#96] [Q&A] [answered]
I just installed beaver's crafting, and it is pretty awesome, but I expected it to have at least some recipes since it had specific system support.  I am in Pf1e.  Have I done something wrong, looking in the wrong place, or should it be empty?  Thanks!

**Comment by AngryBeaver** [comment, marked as answer]
Short Answer: I am sry no. but i hope the system is simple enough to just create some.

Long Answer
Beavers-crafting itself should be system independent. And rely for it on an system interface module. Which then relies on system dependent adaption layers. But the system dependent adaption layer are again decoupled from which module it is used for it can enable all modules that depend on system interface. So the module that grants you system support does not have anything to do with beavers-crafting in fact it should not. It is meant to keep it simple stupid and just enable other modules to work on that system.

I have a module beavers-potions for dnd5e that has 50 potions and on top recipes and ingridents for it. But i wish i had never done this as supporting this is terrible. Keeping the potions up to date with the system and also with other modules they are based on ... terrible idea that :-) So noway i will do this again for any other system.

I have also a handfull mushroom ingredients that do nothing but has names so that would be at least something sadly all packs are system dependend and so also this does only work for dnd5e as every system has its own item-types its not 1 to 1 copy able.

**Reply by R0gu3P403n1x** [reply]
If it is supposed to be empty, then cool.  It is super easy to use.  A little time consuming to add in all of the basic crafting materials, but I understand your standpoint too.  Thanks for the great module!
